packages/cms-cli/commands/init.js

```diff
@@ -93,20 +96,36 @@ function initializeConfigCommand(program) {
     .description(
       `initialize ${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME} for a HubSpot portal`
     )
+    .option(
+      `--${AUTH_METHODS.oauth.value}`,
+      `use ${AUTH_METHODS.oauth.name} for authType`
+    )
+    .option(
+      `--${AUTH_METHODS.api.value}`,
+      `use ${AUTH_METHODS.api.name} for authType`
+    )
```

Comment:
Let's start with making `usertoken` an optional flow and keep the existing initialization flow as the default.